kodi_addon_checker/check_addon.py

```diff
@@ -302,3 +306,19 @@ def _check_file_whitelist(report: Report, file_index, addon_path):
 def relative_path(file_path):
     path_to_print = file_path[len(REL_PATH):]
     return ".{}".format(path_to_print)
+
+
+def _check_complex_files(report: Report, addon_path, max_line_count):
+
+    addon_xml_path = os.path.join(addon_path, "addon.xml")
+    tree = xml.etree.ElementTree.parse(addon_xml_path).getroot()
+    complex_files = []
+
+    for i in tree.findall("extension"):
+        library = i.get("library")
```

Comment:
This won't work for context menu items library is not an attribute of the extension element but an attribute for the item child element

Reply:
For this we'll have to get its children item so i'll have to make another iterator for it right ? because for other this thing works fine.